src/plugins/lan/lan.c

```diff
@@ -338,7 +338,7 @@ ipmi_lan_recv_packet(struct ipmi_intf * intf)
  * asf.data[f:a]= 0x000000000000
  */
 static int
-ipmi_handle_pong(struct ipmi_intf * intf, struct ipmi_rs * rsp)
+ipmi_handle_pong(struct ipmi_rs * rsp)
```

Comment:
Asterisks should stick to the variable name as per [Coding Standards](https://github.com/ipmitool/ipmitool/wiki/Coding-Standards#pointers). I know it wasn't you who put it this way, but once you touch the line, please fix it. Applies everywhere.